gui-tk/wiring.tcl

```diff
@@ -629,7 +629,7 @@ proc contextMenu {x y X Y} {
                 renameVariableInstances
             }
             .wiring.context add command -label "Edit" -command "editItem"
-            .wiring.context add command -label "Copy" -command "canvas.copyItem"
+            .wiring.context add command -label "Copy" -command "copy.canvasItem"
```

Comment:
Where is copy.canvasItem defined?

Reply:
Thanks for catching this. It must have occurred when I tested minsky.paste here instead of copy.canvasItem which was originally there.